### PR TITLE
Add logos

### PR DESCRIPTION
**Pull request by witmicko**
Using b64 images

image

### CODE REVIEW

File: cmd/file-server/main.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	//"os"
```
Comment (JameelB) [inline cmd/file-server/main.go]: Remove commented code

File: cmd/file-server/main.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"log"
```
Comment (JameelB) [inline cmd/file-server/main.go]: We use `github.com/sirupsen/logrus` for logs. I'm not sure if that log there will show up.

File: tmp/build/broker/Dockerfile [hunk 3]
```diff
@@ -1,8 +1,22 @@
 FROM alpine:3.6
 
-RUN adduser -D managed-services
-USER managed-services
+#RUN adduser -D managed-services-broker
```
Comment (JameelB) [inline tmp/build/broker/Dockerfile]: I don't think these are needed. Can probably delete these? @philbrookes

File: static-files/converted-to-b64.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+// red-hat-jboss-fuse
```
Comment (philbrookes) [inline static-files/converted-to-b64.txt:1]: Why is this file included?

File: static-files/eclipse-che-logo.svg [hunk 5]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (philbrookes) [inline static-files/eclipse-che-logo.svg:1]: This one doesn't appear to be in use either?

Comment (witmicko) [reply static-files/eclipse-che-logo.svg:1]: @philbrookes  it was used to generate b64, I think we should keep those logos here alongside b64 master file so it's easier to track changes and reuse.
Although - I'm ok to delete them if you don't want that bundled here

Comment (philbrookes) [reply static-files/eclipse-che-logo.svg:1]: it's ok to keep them if they are expected, I was just making sure it wasn't an accidentally commited file.

**Review by philbrookes — APPROVED** [review]
